packages/components/package.json

```diff
@@ -17,8 +17,8 @@
     "build:wrappers": "yarn build:wrappers:generateWrappers && yarn build:wrappers:generateDSRComponents && yarn build:wrappers:bundleStyles && yarn build:wrappers:bundleUtils",
```

Comment:
Looks like changelog entry is missing.  
Once added, you can update storefront's `sitemap.json` with the url to this PR, too.